tests/ml_monitoring/test_model_staleness.py

```diff
@@ -0,0 +1,55 @@
+import pytest
+import os
+import boto3
+from botocore import UNSIGNED
+from botocore.client import Config
+from datetime import datetime
+
+from src.models.get_model import main as get_model
+
+
+MAX_MODEL_AGE = 60 # Maximum age of model in days before considered stale
+
+
+@pytest.fixture(scope="session")
+def setup_model():
+    # Download the model
+    get_model()
+
+    # Yield to run the tests
+    yield
+
+    # Teardown the model
+    os.remove(os.path.join('models', 'trained_model.joblib'))
```

Comment:
Is think this method is obsolete, in the test you don't use the model file but directly check the bucket to find the model age.
Also if you do need it, consider pulling the features and data branch as that already has this method (current scope module) which could be changed to session.